integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/SparkStoreCreatorForPresto.scala

```diff
@@ -0,0 +1,385 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.integration.spark.testsuite.dataload
+
+import java.io.{File, PrintWriter}
+import java.util.UUID
+
+import scala.util.Random
+
+import org.apache.commons.io.FileUtils
+import org.apache.spark.sql.CarbonEnv
+import org.apache.spark.sql.test.util.QueryTest
+import org.scalatest.BeforeAndAfterAll
+
+import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.datastore.impl.FileFactory
+import org.apache.carbondata.core.util.{CarbonProperties, CarbonUtil}
+import org.apache.carbondata.core.util.path.CarbonTablePath
+import org.apache.carbondata.sdk.file.{CarbonSchemaReader, CarbonWriterBuilder}
+
+class SparkStoreCreatorForPresto extends QueryTest with BeforeAndAfterAll{
+
+  private val timestampFormat = CarbonProperties.getInstance()
+    .getProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT)
+  private val dateFormat = CarbonProperties.getInstance()
+    .getProperty(CarbonCommonConstants.CARBON_DATE_FORMAT)
+  private val rootPath = new File(this.getClass.getResource("/").getPath
+                                  + "../../../..").getCanonicalPath
+  private val sparkStorePath = s"$rootPath/integration/spark/target/spark_store"
+
+  val storePath = storeLocation
+
+  override def beforeAll: Unit = {
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_DATE_FORMAT,
+        CarbonCommonConstants.CARBON_DATE_DEFAULT_FORMAT)
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT,
+        CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT)
+    sql("drop database if exists presto_spark_db cascade")
+    sql("create database presto_spark_db")
+    sql("use presto_spark_db")
+     CarbonUtil.deleteFoldersAndFiles(FileFactory.getCarbonFile
+        (s"$sparkStorePath"))
+  }
+
+  override def afterAll: Unit = {
+    if (null != dateFormat) {
+      CarbonProperties.getInstance()
+        .addProperty(CarbonCommonConstants.CARBON_DATE_FORMAT, dateFormat)
+    }
+    if(null != timestampFormat) {
+      CarbonProperties.getInstance()
+        .addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, timestampFormat)
+    }
+    val source = s"$rootPath/integration/spark/target/warehouse/presto_spark_db.db/"
+    val srcDir = new File(source)
+
+    // Presto will later use this store path to query
+    val destination = s"$rootPath/integration/spark/target/spark_store/"
+    val destDir = new File(destination)
+    FileUtils.copyDirectory(srcDir, destDir)
+    FileUtils.deleteDirectory(srcDir)
+    sql("drop table update_table")
```

Comment:
please make it drop table if exist, so that devloper can run individual testcase

Reply:
done